$NetBSD: patch-aa,v 1.9 2013/05/10 14:37:42 obache Exp $

* killed -DG_DISABLED_DEPRECATED, so msg_gconf_gio will be built even with
  gio-2.0>2.27.1
* add RPATH for samba
*

--- configure.orig	2011-03-21 16:02:39.000000000 +0000
+++ configure
@@ -15633,8 +15633,8 @@ fi
 
 msg_gconf_gio=no
 if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gio-2.0 <= 2.27.1\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "gio-2.0 <= 2.27.1") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gio-2.0\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "gio-2.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
@@ -16928,7 +16928,7 @@ $as_echo "#define HAVE_SAMBA /**/" >>con
 
 		msg_samba="yes"
                 if test x$with_samba_libs != x/usr/lib; then
-                        SAMBA_LIBS="-L$with_samba_libs -lsmbclient"
+                        SAMBA_LIBS="-L$with_samba_libs -lsmbclient -Wl,-R$with_samba_libs"
                 else
                 	SAMBA_LIBS="-lsmbclient"
                 fi
@@ -17376,7 +17376,7 @@ _ACEOF
 fi
 
 
-if test "$ac_cv_func_statfs" = yes ; then
+if test "$ac_cv_func_statvfs" != yes -a "$ac_cv_func_statfs" = yes ; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking number of arguments to statfs()" >&5
 $as_echo_n "checking number of arguments to statfs()... " >&6; }
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
